app/login/controller.js

```diff
@@ -2,6 +2,10 @@ import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
+import Auth0Lock from 'auth0-lock';
+
+const clientId = 'YMyNn8AN7EkZQjAy8WHbX9Jc4LrYwtB4';
+const domain = 'dev-4pbsyp02.us.auth0.com';
```

Comment:
Consider to move these variables to `.env` file - more here: https://github.com/fivetanley/ember-cli-dotenv